mix.exs

```diff
@@ -60,7 +60,11 @@ defmodule Tower.MixProject do
       # Test
       {:assert_eventually, "~> 1.0", only: :test},
       {:plug_cowboy, "~> 2.7", only: :test},
-      {:bandit, "~> 1.5", only: :test},
+      {
+        :bandit,
+        "~> 1.5",
+        only: :test, github: "mimiquate/bandit", branch: "throwing-plug", override: true
```

Comment:
Temporarily pointing to https://github.com/mtrudel/bandit/pull/411 to test how it would work here.